indexclient/client.py

```diff
@@ -291,7 +291,11 @@ def create(
             data=json_dumps(json),
             auth=self.auth,
         )
-        return Document(self, resp.json()["did"])
+        try:
+            return Document(self, resp.json()["did"])
+        except KeyError as e:
+            logging.error(resp.json())
```

Comment:
is it better to check the status of the response instead? if its successful, then its safe to assume there is an `id` in the payload